app/services/sufia/user_stat_importer.rb

```diff
@@ -46,81 +54,93 @@ def sorted_users
 
     private
 
-      def delay
```

Comment:
Why are you changing the indentation here?  Please revert this. We have been using the style where you "Indent after private/protected" http://guides.rubyonrails.org/contributing_to_ruby_on_rails.html#follow-the-coding-conventions